### PR TITLE
Payment integration

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**

### CODE REVIEW

File: src/pages/profile/profile.css [hunk 1]
```diff
@@ -4,4 +4,12 @@
...
 .profile-page-container {
   padding: 2rem;
+  margin-top: 4rem;
+  border: 1px solid var(--custom-pri-clr);
+}
+
+.btn-small {
+  padding: 0.5rem 0.7rem;
+  font-size: 0.8rem;
+  outline: none;
```
Comment (gauravsinhaweb) [inline src/pages/profile/profile.css:14]: give some outline for accessibility!

File: src/pages/address/Address.jsx [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+
+    setAddressInfo({ ...addressInitialState, isAddress: false });
+    setIsDisabled(true);
+  }
+
+  return (
+    <div>
+      <Navbar />
+
+      <section className="d-flex justify-cont-center address-form-container">
```
Comment (gauravsinhaweb) [inline src/pages/address/Address.jsx:86]: suggestion: you can break the display part in another component, for readability

**Review by gauravsinhaweb — APPROVED** [review]
couldn't find any major issue, UI looks good, you are good to merge

**Review by saicharan-gouru — APPROVED** [review]
Looks good to merge

**Pull request merged**